spec/virtual_total_spec.rb

```diff
@@ -245,7 +271,7 @@ class Author < VitualTotalTestBase
       m2 = model_with_children(2)
       mc = m1.class
       expect {
-        ms = mc.select(:id, mc.arel_attribute(:total_books).as("total_books"))
```

Comment:
Huh... why was this here in the first place?

Reply:
ikr?

a long time ago (over 2 years?) we were not smart enough to support `select(:virtual_attribute)`